aten/src/ATen/native/native_functions.yaml

```diff
@@ -2383,11 +2383,6 @@
   dispatch:
     CompositeExplicitAutograd: kl_div
 
-- func: kl_div_backward(Tensor grad_output, Tensor self, Tensor target, int reduction=Mean, *, bool log_target=False) -> Tensor
```

Comment:
I don't think anyone is using this but this is, of course, a BC-breaking change.